data/scenarios/Challenges/Ranching/gated-paddock.yaml

```diff
@@ -311,6 +312,10 @@ objectives:
           end;
 
       anySheep (has "clover") 3;
+    id: feed_sheep
```

Comment:
As a reader I would prefer having the id first:
```YAML
- id: feed_sheep
  goal:
    - ...
```
This way it kind of merges with the next thing. Also, it does not work well with document folding.